README.md

```diff
@@ -47,6 +47,7 @@ To get a quick understanding about ESP-IDF and Eclipse plugin features check our
 * [ Application Level Tracing ](#appLvlTracing)<br>
 * [ ESP-IDF master update](#updateEspIdfMaster)<br>
 * [ Changing Language ](#changeLanguage)<br>
+* [ Suggestions for solving errors from ESP-IDF by hints viewer](#hintsViewer)<br>
```

Comment:
Could we add this as a subsection in the [troubleshooting](https://github.com/espressif/idf-eclipse-plugin#troubleshooting) guide instead of here?

Reply:
Hi @kolipakakondal,
thanks for the review. Moved this section as suggested